spec/requests/admin/configs_spec.rb

```diff
@@ -610,7 +610,6 @@
           expect(SiteConfig.primary_brand_color_hex).not_to eq(hex)
         end
 
-
```

Comment:
@ludwiczakpawel can you remove this file as it's not related to the PR?

Reply:
Done. I think you need to re-review :)